lib/ama_layout/decorators/navigation_item_decorator.rb

```diff
@@ -24,7 +24,13 @@ def active_class
 
   private
     def active_link?
-      sub_nav.map(&:link).push(link).include? current_url
+      sub_nav.map(&:link).push(link).include? current_url_without_query
+    end
+
+    def current_url_without_query
+      URI.parse(current_url).tap {|uri| uri.query = nil }.to_s
```

Comment:
What do you think about adding an extra space before the block variable `uri` here?